static/src/javascripts/projects/common/utils/detect.js

```diff
@@ -235,7 +235,8 @@ define([
             });
         };
 
-        var woff2Candidacy = /(chrome|firefox)\/([0-9]+)/.exec(ua);
+        var excludeEdge = /^(edge\/([0-9]+))/.test(ua);
```

Comment:
Oops, dodgy logic, fixed in https://github.com/guardian/frontend/commit/92721aa8fa7d9d4e39d0727db08ea424aa2d1009